README.md

```diff
@@ -0,0 +1,30 @@
+# Conway's game of life
+
+My implementation of Conway's game of life.
+
+![Conway's game of life](conways.gif)
+
+
+## Rules
+
+- Any live cell with fewer than two live neighbours dies, as if by underpopulation.
+- Any live cell with two or three live neighbours lives on to the next generation.
+- Any live cell with more than three live neighbours dies, as if by overpopulation.
+- Any dead cell with exactly three live neighbours becomes a live cell, as if by reproduction.
+
+(Rules extracted from [Wikipedia](https://en.wikipedia.org/wiki/Conway%27s_Game_of_Life))
+
+## Dependencies
+
+It's built using Rust with the crate macroquad and uses Make to run
+
```

Comment:
This is correct, but I need a bit more. Let's add the relevant make targets to start the game.

Reply:
I saw that you added the make targets needed to be run in the PR's description. Put them in the README too.